proof-systems/src/gm17/verifier.rs

```diff
@@ -54,7 +54,8 @@ pub fn verify_proof<E: PairingEngine>(
             (proof.c.into(), pvk.h_pc.clone()),
         ]
         .iter(),
-    );
+    ).unwrap();
```

Comment:
Why we are unwrapping here ? We have already the Result as a return type so let's just put "?"